applications/luci-app-transmission/luasrc/controller/transmission.lua

```diff
@@ -8,7 +8,6 @@ function index()
 		return
 	end
 
-	local page = entry({"admin", "services", "transmission"}, cbi("transmission"), _("Transmission"))
```

Comment:
Since the only remaining controller action is providing menu entries, let's switch to JSON menu entries while we're at it.
Create an ` applications/luci-app-transmission/root/usr/share/luci/menu.d/luci-app-transmission.json` with the following content:

```
{
	"admin/services/transmission": {
		"title": "Transmission",
		"action": {
			"type": "view",
			"path": "transmission"
		}
	}
}
```

Afterwards you can delete the Lua controller file completely.